### ISSUE TITLE
Add No Randomize Layout attribute to regression test

### ISSUE THREAD

**Issue by tim-pugh** [clang-randstruct/llvm-project#37]
Since we've merged in the new attribute we will need to add it to the regression test.

**Comment by tim-pugh** [comment]
Closing this issue. I just ran the regression test suite `make clang-test` and it passed without error.

This is rather odd considering we added the second attribute but we'll re-run it at a later date. It may be because it's not being used properly quite yet.

**Comment by connorkuehl** [comment]
> This is rather odd considering we added the second attribute but we'll re-run it at a later date. It may be because it's not being used properly quite yet.

I was confused about this this morning too, but @Jafosterja mentioned this code was reverted.

Searching the codebase for "Randomize Layout" confirms this (since "No Randomize Layout" matches "Randomize Layout")

```
connor@mead-garden:~/src/git/llvm-project/clang$ ag -i randomizelayout
test/Misc/pragma-attribute-supported-attributes-list.test
120:// CHECK-NEXT: RandomizeLayout (SubjectMatchRule_record)

lib/AST/RecordLayoutBuilder.cpp
2991:  bool ShouldBeRandomized = D->getAttr<RandomizeLayoutAttr>() != nullptr;

lib/Sema/SemaDeclAttr.cpp
6856:  case ParsedAttr::AT_RandomizeLayout:
6857:    handleSimpleAttribute<RandomizeLayoutAttr>(S, D, AL);

include/clang/Basic/Attr.td
3205:def RandomizeLayout : InheritableAttr {
```

**Comment by tim-pugh** [comment]
https://github.com/clang-randstruct/llvm-project/pull/27/commits/f06d4cc861b166735a8b1e57faf287e4e7ea7c11

**Comment by tim-pugh** [comment]
We can pull those in now, but from what I'm seeing, they were removed. Does this make sense?

**Comment by tim-pugh** [comment]
https://github.com/clang-randstruct/llvm-project/commit/d95e6247f5ddd2812f03bd3a829d9299b751abfe

This was the revert commit

**Comment by connorkuehl** [comment]
> This was the revert commit

Yeah, that's the source of my confusion. I saw that those commits I linked earlier were merged in to `develop` but I couldn't find where they were taken out.

**Comment by connorkuehl** [comment]
Closed by #39
